engine/table/src/test/java/io/deephaven/engine/table/impl/QueryTableFlattenTest.java

```diff
@@ -86,93 +84,86 @@ public void testLegacyFlatten3() {
         final QueryTable queryTable =
                 TstUtils.testRefreshingTable(i(data).toTracking(), longCol("intCol", data));
 
-        final TestHelper helper = new TestHelper<>(queryTable.flatten(), SimpleShiftObliviousListener::new);
+        final TestHelper<?> helper = new TestHelper<>(queryTable.flatten(), SimpleShiftObliviousListener::new);
 
         helper.modAndValidate(() -> {
             addToTable(queryTable, i(70, 71, 78, 81), longCol("intCol", 70, 71, 78, 81));
             queryTable.notifyListeners(i(70, 71, 78, 81), i(), i(24, 46, 74, 100, 104));
         }, indexByRange(10, 13), i(), indexByRange(2, 9));
     }
 
-    @Test
     public void testLegacyFlattenModifications() {
         QueryTableTest.testLegacyFlattenModifications(arg -> (QueryTable) arg.flatten());
     }
 
-    @Test
```

Comment:
You might just make everything a new junit instead of taking away @Test annotations; are we sure these still run?

Reply:
I double checked that the tests do run. (One of the flatten tests was a cause of one of the CI failures, too.) I've filed #3272 to track upgrading past junit4.